src/shogun/multiclass/KDTreeKNNSolver.h

```diff
@@ -16,10 +16,9 @@ namespace shogun
 {
 
 /**
- * The KDTREEKNNSolver class is a solver class which inherit from KNNSolver class.
- * It use a k-d tree (short for k-dimensional tree) to delivery the nearest neighbor searches
- * involving the multidimensional search key.
- * Check https://en.wikipedia.org/wiki/K-d_tree for more detail about K-D tree.
+ * KD-tree solver. It uses k-d tree (short for k-dimensional tree) to speed up the 
+ * nearest neighbour computation. involving the multidimensional search key.
```

Comment:
Sentences start with capital letter. The second sentence misses a subject. I also dont understand what that is supposed to mean